lib/Sema/SemaResolveType.cpp

```diff
@@ -483,13 +483,44 @@ void TypeResolver::visit(CallExprType &CE) {
       case Identifier::SpecialIdentifier::abi_encodePacked:
       case Identifier::SpecialIdentifier::abi_encode:
         break;
-      case Identifier::SpecialIdentifier::abi_encodeWithSelector:
+      case Identifier::SpecialIdentifier::abi_encodeWithSignature: {
+        auto &Arg0 = CE.getRawArguments().at(0);
```

Comment:
Does the `Arg0` have a specific name in the spec?

Reply:
change `Arg0` to `Signature`